components/ai_chat/core/browser/engine/engine_consumer_llama_unittest.cc

```diff
@@ -61,20 +61,22 @@ TEST_F(EngineConsumerLlamaUnitTest, TestGenerateAssistantResponse) {
   auto* mock_remote_completion_client =
       static_cast<MockRemoteCompletionClient*>(engine_->GetAPIForTesting());
   std::string prompt_before_time_and_date =
-      "<s>[INST] <<SYS>>\nThe current time and date is ";
+      "<s>[INST] The current time and date is ";
```

Comment:
FYI this unit test only covers mixtral (see SetUp), we should create new test cases for llama now they've diverged.